content/source/docs/enterprise/private/faq.html.md

```diff
@@ -183,6 +183,9 @@ from the ELB is routed. Many other internal Private Terraform Enterprise service
 but they do not require external traffic. The AWS security group for the
 instance as well as software firewall rules within the runtime enforce this.
 
+Additionally, the port ranges **9870-9880** and **23000-23100** must be accessible to the host and
+the subnet it is on. These ports are used for internal communication between internal services.
+
```

Comment:
This has been added in the AMI section, but I think we want it up in the general section, if anywhere, as this change won't impact the AMI, right?

Reply:
Oh ack. yes. Thank you.

Reply:
👍